code/game/objects/items/weapons/scissors.dm

```diff
@@ -119,9 +119,7 @@
 				user.visible_message("<span class='danger'>[user] abruptly stops cutting [M]'s hair and slices their throat!</span>", "<span class='danger'>You stop cutting [M]'s hair and slice their throat!</span>") //Just a little off the top.
 				H.AdjustLoseBreath(10) //30 Oxy damage over time
 				H.apply_damage(18, BRUTE, "head", sharp =1, edge =1, used_weapon = "scissors")
-				var/turf/location = get_turf(H)
-				if(istype(location, /turf/simulated))
-					H.add_splatter_floor(location)
+				add_mob_blood(H)
```

Comment:
I didn't delete my comment in time.

I was incorrect on this, what you had is correct, but should be changed too:

```
		var/turf/location = get_turf(src)
		L.add_splatter_floor(location)
```